spec/factory_girl/votes.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :vote do
+    value 1
+    votable_id 1
+    votable_type "MyString"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.